theano/tensor/nnet/tests/test_abstract_conv.py

```diff
@@ -227,7 +230,8 @@ def run_gradinput(self, inputs_shape, filters_shape, output_shape,
                     border_mode=border_mode, subsample=subsample,
                     conv_mode=conv_mode)
         f = theano.function([], c, mode=mode)
-        self.assertTrue(hasattr(f.maker.fgraph.outputs[0].tag, 'trace'))
+        if check_trace:
+            self.assertTrue(check_stack_trace(f, ops_to_check='all'))
```

Comment:
There are subtensors (`[:, :, ::-1, ::-1]`) inserted to flip kernels and such, and those may not have an obvious equivalent, so they have no stack trace.
I think it would make sense to put `target_op` in `ops_to_check`, and maybe put that inside the `if target_op is not None:` line below.